### PR TITLE
(PC-22882)[PRO] refactor: clean disability icon / home icon / hourgla…

### PR DESCRIPTION
**Pull request by smokhtari-passculture**
Lien vers le ticket Jira : https://passculture.atlassian.net/browse/PC-22882

## But de la pull request

Clean les icônes : 

mental-disability
motor-disability
visual-disability
ico-home
ico-hourglass

### CODE REVIEW

File: pro/src/pages/AdageIframe/app/components/OffersInstantSearch/OffersSearch/Offers/PrebookingButton/PrebookingButton.tsx [hunk 1]
```diff
@@ -3,7 +3,7 @@ import React, { useCallback, useState } from 'react'
 
 import { OfferStockResponse } from 'apiClient/adage'
 import { apiAdage } from 'apiClient/api'
-import { ReactComponent as HourGlassIcon } from 'icons/ico-hourglass.svg'
+import StrokeHourglass from 'icons/stroke-hourglass.svg'
```
Comment (AmineLouveau) [inline pro/src/pages/AdageIframe/app/components/OffersInstantSearch/OffersSearch/Offers/PrebookingButton/PrebookingButton.tsx:6]:
```suggestion
import strokeHourglass from 'icons/stroke-hourglass.svg'
```

File: pro/src/pages/Offers/Offers/OfferItem/Cells/CollectiveOfferStatusCell/CollectiveOfferStatusCell.tsx [hunk 2]
```diff
@@ -5,12 +5,13 @@ import { OfferStatus } from 'apiClient/v1'
 import { CollectiveStatusLabel } from 'components/CollectiveStatusLabel'
 import { Offer } from 'core/Offers/types'
 import { ReactComponent as StatusEndedIcon } from 'icons/ico-double-check.svg'
-import { ReactComponent as StatusPreBookedIcon } from 'icons/ico-hourglass.svg'
 import { ReactComponent as StatusExpiredIcon } from 'icons/ico-status-expired.svg'
 import { ReactComponent as StatusInactiveIcon } from 'icons/ico-status-inactive.svg'
 import { ReactComponent as StatusPendingIcon } from 'icons/ico-status-pending.svg'
 import { ReactComponent as StatusRejectedIcon } from 'icons/ico-status-rejected.svg'
 import { ReactComponent as StatusValidatedIcon } from 'icons/ico-status-validated.svg'
+import StrokeHourglass from 'icons/stroke-hourglass.svg'
```
Comment (AmineLouveau) [inline pro/src/pages/Offers/Offers/OfferItem/Cells/CollectiveOfferStatusCell/CollectiveOfferStatusCell.tsx:13]:
```suggestion
import strokeHourglass from 'icons/stroke-hourglass.svg'
```

File: pro/src/components/Header/Header.jsx [hunk 3]
```diff
@@ -6,12 +6,12 @@ import { Events } from 'core/FirebaseEvents/constants'
 import useActiveFeature from 'hooks/useActiveFeature'
 import useAnalytics from 'hooks/useAnalytics'
 import { CalendarIcon } from 'icons'
-import { ReactComponent as IconHome } from 'icons/ico-home.svg'
 import { ReactComponent as IconOffers } from 'icons/ico-offers.svg'
 import { ReactComponent as IconSignout } from 'icons/ico-signout.svg'
 import { ReactComponent as StatsIcon } from 'icons/ico-stats.svg'
 import deskIcon from 'icons/stroke-desk.svg'
 import strokeEuro from 'icons/stroke-euro.svg'
+import StrokeHome from 'icons/stroke-home.svg'
```
Comment (AmineLouveau) [inline pro/src/components/Header/Header.jsx:14]:
```suggestion
import strokeHome from 'icons/stroke-home.svg'
```

**Review by AmineLouveau — APPROVED** [review]
Juste garder la convention de nommage à garder. Sinon RAS.

**Review by hugocapuano — APPROVED** [review]

**Pull request merged**